assign_rights/serializers.py

```diff
@@ -28,26 +28,58 @@ class RightsShellSerializer(serializers.ModelSerializer):
     """
     start_date = serializers.CharField()
     end_date = serializers.CharField()
-    status = serializers.CharField(source="copyright_status")
-    terms = serializers.CharField(source="license_terms")
-    citation = serializers.CharField(source="statute_citation")
     rights_granted = serializers.ListField(default=[])
-    jurisdiction = serializers.SerializerMethodField()
 
     class Meta:
         model = RightsShell
         fields = (
             "rights_basis",
-            "determination_date",
-            "jurisdiction",
             "start_date",
             "end_date",
             "note",
-            "status",
-            "terms",
-            "citation",
             "rights_granted"
         )
 
+
+class OtherSerializer(RightsShellSerializer):
+    rights_basis = serializers.SerializerMethodField()
+    other_basis = serializers.SerializerMethodField()
```

Comment:
You can do this much more simply: 
```suggestion
    other_basis = serializers.CharField(source='rights_basis')
```

https://www.django-rest-framework.org/api-guide/serializers/#specifying-fields-explicitly